soccer/config/realEindhoven.xml

```diff
@@ -1,26 +1,28 @@
 <config>
```

Comment:
Let's revert realEindhoven.xml to last year's version and save our changes as a new realChina.xml file.